components/expressvars/package.json

```diff
@@ -10,7 +10,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/adobe/spectrum-css.git",
-    "directory": "components/vars"
+    "directory": "components/expressvars"
```

Comment:
@pfulton Okay this is totally unrelated but I was in the file and noticed this and thought, that perhaps needs fixing.  Can open separate PR if you like!

Reply:
Wow, great catch. Thanks for fixing!

I think we can sneak this in with this PR, but, if possible: can you keep this change in its own commit, and it can be a `fix` or a `refactor` - it's probably more of a `fix`...but either way, it's just a patch.

THANKS.

Reply:
Awesome - just pushed up that update.